make-base/utils.mk

```diff
@@ -0,0 +1,99 @@
+.PHONY: always
+
+# Function that find project directories by a file that must exist inside of it
+define find_projects_containing
+$(sort \
```

Comment:
So, this function more or less defines what a `project directory` is.

Basically, a project directory is any directory referenced as a docker image that exists either in the top level directory, docker directory, swarm directory, or etna/packages directory.

If we want to add new project roots, this is the function to adjust.